src/cat-data-service/src/legacy_service/health.rs

```diff
@@ -27,6 +50,54 @@ async fn live_exec() -> Result<bool, Error> {
     Ok(true)
 }
 
+async fn retry_after_exec(Query(params): Query<RetryAfterParams>, state: Arc<State>) -> Response {
```

Comment:
Not quite clear why this endpoint is needed, can you provide some documentation pls

Reply:
This endpoint is used to configure the `retry-after` header, that can be a datetime string or a positive integer, in 503 responses. This is useful to inform other services when they should expect to retry connection to our service.